subsys/shell/shell_fprintf.c

```diff
@@ -32,7 +32,7 @@ static int out_func(int c, void *ctx)
 void shell_fprintf_fmt(const struct shell_fprintf *sh_fprintf,
 		       const char *fmt, va_list args)
 {
-#ifndef CONFIG_NEWLIB_LIBC
```

Comment:
Commit message for this change:

> The posix arch does not compile neither of Zephyr's libc,

sounds a bit strange in the standard English, should be "The posix arch does not compile *either* of Zephyr's libc's,"